### ISSUE TITLE
Bug: Player markers

### ISSUE THREAD

**Issue by Marcel1802** [A3Wasteland/ArmA3_Wasteland.Altis#335]
When players created a group the player markers only show 1 member of the group

**Comment by LouDnl** [comment]
This is because the player markers only shows the group leader. It's still something from the arma 2 version of Wasteland I believe. I asked Rev a long time ago about this too. It should actually be removed.

**Comment by Gundeamon** [comment]
Hi Sorry about this but i think t5his is the same topic, in game there are blue markers above team mates, UGVs and the like, if your Blufor it is a blue two tone diamond shape, red fro red etc, is there a way of removing this from the game as it is a bit rubbish when my team mate is running to a position near me and all i can see is a big blue square, thanks.

**Comment by LouDnl** [comment]
No it's something different. Marcel1802 was talking about the show player makers option in the admin menu.

On the otherhand i'm also wondering how to disable the green/red/blue squares in the players octagon.

**Comment by AgentRev** [comment]
> disable the green/red/blue squares in the players octagon

I don't get it.

**Comment by osuapoc** [comment]
They're talking about turning off the friendly player markers.  You can reduce the size of the markers.  Search my commits, LouD, for how I did that.  I'm on my phone so my access is limited atm.

-----Original Message-----

**Comment by LouDnl** [comment]
The ones you see inside the hexagons in this video. https://www.youtube.com/watch?v=j-hJQwZceWk

**Comment by AgentRev** [comment]
Oh so what you mean is, you wish to disable icons for people in your group but not others?

**Comment by Gundeamon** [comment]
Well I don't mind disabling all of them, its just so unrealistic having these blue markets floating about in game.
--
Sent from myMail app for Android Friday, 07 August 2015, 04:49pm +01:00 from Agent Revolution < notifications@github.com> :

**Comment by AgentRev** [comment]
I'm talking to LouD.

**Comment by LouDnl** [comment]
Players on my servers have been bugging me if I could disable the colored markers or make them smaller. So no, not disable them. The hexagon markers should stay.

**Comment by Gundeamon** [comment]
Perhaps an option to do so, or does anyone have the script to do so, bearing in mind I run a private game for me and some friends, its definitely not public.
--
Sent from myMail app for Android Friday, 07 August 2015, 04:53pm +01:00 from LouDnl < notifications@github.com> :

**Comment by AgentRev** [comment]
@LouDnl: Hexagons are only shown for group members and are controlled by the engine. Are you saying some people do not want to know if somebody's friendly?

@Gundeamon: To disable player icons permanently, comment out [this](https://github.com/A3Wasteland/ArmA3_Wasteland.Altis/blob/a6d06946616920d6d31eb89eb3a156923aa30e94/client/init.sqf#L137) line

**Comment by LouDnl** [comment]
No its the Heaxgon they want :) It's the PlayerIcons that are too large (according to some of our players). But since you pointed @Gundeamon in the right direction, I will be able to figure this out too. Thanks.

**Comment by LouDnl** [comment]
Changed it for our servers :)
https://github.com/LouDnl/ArmA3_Wasteland.Altis/commit/2e244a0badfd2a06714bbf2e058d8164852fd7bc

**Comment by Marcel1802** [comment]
I don't mean the markers which show your mates i mean the markers from the admin menu

**Comment by AgentRev** [comment]
You should've commented just the value

    #define ICON_fadeDistance 1000 // default: 1250

**Comment by AgentRev** [comment]
@Marcel1802 yes we know, the conversation sorta drifted away. The particular feature you were talking about is provided by the Arma 3 engine, all the admin panel option does is enable it. Its purpose is only to show group leaders, and that can't be changed.
